CIPs/CIP-0037.md

```diff
@@ -0,0 +1,49 @@
+---
+cip: ?
+title: Unambiguous name for ERC-20 on Celo chain
+author: aaron deruvo, @aaronmgdr, Martin Chrzanowski, etc 
```

Comment:
Need Martin's email or github handle, whichever is more preferable.

Reply:
added